### PR TITLE
More adjustments to programs/services list. Resolves #202.

### LINKED ISSUE #202: Leadership development monthly meetings
Should move to Community Engagement And Organizing program.

### PR DESCRIPTION
**Pull request by joeldenning**
See #202

### CODE REVIEW

File: migrations/sqls/20190806200938-issue-202-down.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+OR serviceName = "I-821 Temporary Protected Status (TPS)"
+OR serviceName = "I-765 Application for Employment Authorization"
+OR serviceName = "Freedom of Information Act (FOIA)"
+OR serviceName = "Status Check"
+OR serviceName = "I-94 Request"
+OR serviceName = "Family Petition - Consular Processing";
+
+UPDATE services
+SET serviceName = "Family Petition"
+WHERE serviceName = "Family Petition - Adjustment of Status";
```
Comment (MeghanBomberger) [inline migrations/sqls/20190806200938-issue-202-down.sql]: The "no new line at end of file" indicator popped up here.

File: migrations/sqls/20190806200938-issue-202-up.sql [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  ("General Consultation", "General Consultation",
+    (SELECT id FROM programs WHERE programName = "Immigration")
+  ),
+  ("Translation", "Translation", (
+    SELECT id FROM programs WHERE programName = "Immigration"
+  )),
+  ("Green Card Renewal", "Green Card Renewal", (
+    SELECT id FROM programs WHERE programName = "Immigration"
+  )),
+  ("I-821 Temporary Protected Status (TPS)", "I-821 Temporary Protected Status", (
```
Comment (MeghanBomberger) [inline migrations/sqls/20190806200938-issue-202-up.sql]: Should the second be "I-821 Temporary Protected Status (TPS)" also for consistency?

File: migrations/sqls/20190806200938-issue-202-down.sql (hunk 1 above)
Comment (joeldenning) [reply migrations/sqls/20190806200938-issue-202-down.sql]: That's okay -- new lines at end of file are something that prettier adds to most files, but they aren't really important or even necessarily better. I guess prettier doesn't currently support formatting SQL files, which is why it didn't add the new line to this one.

File: migrations/sqls/20190806200938-issue-202-up.sql (hunk 2 above)
Comment (joeldenning) [reply migrations/sqls/20190806200938-issue-202-up.sql]: Good catch, updated.

**Pull request merged**